library/accessory_workflows/build_cellranger_reference/BuildCellRangerReferenceBundle.wdl

```diff
@@ -0,0 +1,102 @@
+version 1.0
+
+
+workflow BuildCellRangerRef {
+  input {
+    String ref_fasta
+    String gtf_file
+    String ref_name
+  }
+
+  call BuildCellRangerReference {
+    input:
+      ref_fasta = ref_fasta,
+      gtf_file = gtf_file,
+      ref_name = ref_name
+  }
+
+  output {
+    File cellranger_ref = BuildCellRangerReference.cellRangerRef
+  }
+}
+
+task BuildCellRangerReference {
+  input {
+    String ref_fasta
+    String gtf_file
+    String ref_name
+
+    String docker = "quay.io/humancellatlas/secondary-analysis-cellranger:v1.0.0"
+    String memory = "57 GB"
+    Int boot_disk_size_gb = 12
+    Int disk_space = 50
+    Int cpu = 64
+  }
+  parameter_meta {
+    ref_fasta: "Genome reference in fasta format"
+    gtf_file: "Gene annotation file in gtf format"
+    disk_space: "(optional) the amount of disk space (GB) to provision for this task"
+    ref_name: "The expected file name of the output reference"
+    docker: "(optional) the docker image containing the runtime environment for this task"
+    boot_disk_size_gb: "(optional) the amount of space (GB) of boot disk to provision for this task"
+    cpu: "(optional) the number of cpus to provision for this task"
+    memory: "(optional) the amount of memory (MB) to provision for this task"
+  }
+
+  command <<<
+
+    set -euo pipefail
+
```

Comment:
Could you add some comments to summarize the code below? It's a little dense. Looks like it's copying down the file and gunzipping it?